uk.co.compucorp.civicrm.hrcore/backstop_data/casper_scripts/ssp-leave-absences/staff/my-leave-calendar_current-month-visible.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+var page = require('../../../page-objects/ssp-la-my-leave');
+
+module.exports = function (casper) {
+  page.init(casper).waitForCurrentMonthsVisibility();
```

Comment:
if you use the `waitForReady` method (see other comment), this can simply be
```js
page.init(casper);
```
because `init` will automatically call `waitForReady`